codespell_lib/data/dictionary.txt

```diff
@@ -24448,13 +24496,19 @@ finializing->finalizing
 finilizes->finalizes
 finisch->finish, Finnish,
 finisched->finished
+finisches->finishes
+finisching->finishing
 finised->finished
+finishd->finished
 finishe->finished, finish,
 finishied->finished
 finishs->finishes
+finising->finishing
 finitel->finite
 finness->finesse
```

Comment:
`finness` coule be a misspelling of `fineness`, perhaps even `fitness`.

Reply:
`finness->fineness` exists. `finness->fitness` exists but is rarer. I'll add both.